backend/src/auth/entities/auth.entity.ts

```diff
@@ -0,0 +1,22 @@
+import { ObjectType, Field } from '@nestjs/graphql';
+
+@ObjectType()
+export class Auth {
```

Comment:
I would rename this to user — since both class and filename

Since this is essentially a user object

Reply:
Done!